### ISSUE TITLE
Allow custom input

### ISSUE THREAD

**Issue by greatvovan** [wolfgang42/yaml-multiline#11]
I believe it will be very useful to give user an ability to verify the result against custom input. Can you make the input editable (and the result dynamically updated)?

P.S. Thanks for your site, it is impossible to remember all this things, I refer to it pretty often.

**Comment by rufreakde** [comment]
Bump from me as well. An input field that starts from the second line.
```yaml
# YAML
example: |\n
··Several lines of text,\n # input field from here
··with some "quotes" of various 'types',\n
··and also a blank line:\n
··\n
··and some text with\n
····extra indentation\n
··on the next line,\n
··plus another line at the end.\n
··\n
··\n
```

**Comment by wolfgang42** [comment]
Agreed this would be a useful feature, though I’m unlikely to get around to implementing it. I’ll keep the issue open anyway, though, in case it ever does happen someday.
